app/src/main/java/com/bnyro/recorder/ui/MainActivity.kt

```diff
@@ -16,12 +16,13 @@ import com.bnyro.recorder.ui.models.ThemeModel
 import com.bnyro.recorder.ui.theme.RecordYouTheme
 
 class MainActivity : ComponentActivity() {
+    private var initialRecorder = RecorderType.NONE
+    private var exit = false
```

Comment:
```suggestion
    private var exitAfterRecordingStart = false
```
sounds more self-explaining